applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -68,6 +72,17 @@ static const struct bt_data ad_peer[] = {
 	BT_CSIP_DATA_RSI(csip_rsi)
 };
 
+static void le_audio_event_publish(enum le_audio_evt_type event)
+{
+	int ret;
+	struct le_audio_msg msg;
+
+	msg.event = event;
+
+	ret = zbus_chan_pub(&le_audio_chan, &msg, K_NO_WAIT);
+	ERR_CHK(ret);
```

Comment:
Should we return the ret like the ctrl_event module did in the before?

Reply:
Discussed this, having the ERR_CHK in le_audio_event_publish saves much code